### DISCUSSION TITLE
The documentation of vlib should be improved

### DISCUSSION THREAD

**Discussion by chriseaton** [vlang/v#12448] [Ideas]
I really like V, but one thing really stands out to me, and that's the lack of standards for code documentation. Walking through the code of vlib, it's a rarity to find a function, struct, or enum, etc. with any sort of comment documentation. 

This really takes me aback given that V wants to have a rigid format and standard to it's approach and style - yet there's no enforcement on merges to document your code - like it's that hard to write a sentence about the function they've created. 

I feel like if this isn't rectified soon it will continue to hurt the language, and as more is added to vlib undocumented the more of a mess it will become.

**Comment by larpon** [comment]
[PRs are welcome](https://github.com/vlang/v/issues/7047)

**Reply by chriseaton** [reply]
Excellent- thanks for pointing this out. But to go a bit further, what do you think about enforcing code documentation for any PR - perhaps called out explicitly in CONTRIBUTING.md?

**Comment by JalonSolov** [comment]
It's a bit too early for that.  There has been discussion about changing to require `///` to mean documentation vs just `//` for regular comments.  If would be better if that were decided (then implemented, if the change is made) before requiring something specific in PRs.
